src/wizard/Step3/components/CreateApplicationForm.vue

```diff
@@ -5,17 +5,17 @@
         Let’s create your new application
       </h1>
       <p v-if="!isStepTouched">
-        In this section, we will set up your cloud workspaces within
-        {{ this.$store.getters.getPortfolio.csp }}. If you have more than one
-        application, we will walk through them one at a time. Select 
+        {{ this.$store.getters["wizard/getPortfolio"].csp }}. If you have more
```

Comment:
should probably not use the getter directly here - I know this is old code just refactored for the /wizard getter, but it should be a computed value that returns either the CSP from the wizard store, or the phrase "your selected CSP" if no CSP is set. also line 18 below.

Reply:
fixed this with a computed